src/main/java/catan/domain/model/game/GameBean.java

```diff
@@ -290,6 +300,17 @@ public void setHexes(Set<HexBean> hexes) {
         this.hexes = hexes;
     }
 
+    public Set<HexBean> getHexesWithCurrentDiceValue() {
+        Set<HexBean> hexesWithDiceNumber = new HashSet<HexBean>();
+        for (HexBean hex : this.hexes) {
```

Comment:
I think that you iterate values once again, when you return them, so what is the purpose of this method? to remove one "if" check, you have added 10 more new lines of code